src/pages/groupComparison/comparisonGroupManager/ComparisonGroupManager.tsx

```diff
@@ -272,7 +272,7 @@ export default class ComparisonGroupManager extends React.Component<
                         this.props.store.updateComparisonGroupsFilter();
                     }}
                 >
-                    View
+                    Select
```

Comment:
probably should also rename to `selectButton` and look elswhere for `view` to replace it with `select`

Reply:
@jjgao should I rename it to `selectButton`?
@adamabeshouse I haven't found any `view` in group comparison related pages.